workflow101.Rmd

```diff
@@ -311,7 +318,7 @@ In fact, this is a very good reason to use a productivity app in the first place
 It is very normal -- and productive!
 -- to have multiple Projects open at once.
 
-### RStudio Project vs. active usethis project
+### RStudio Project vs. active usethis project {#sec-rstudio-project-vs.-active-usethis-project}
```

Comment:
```suggestion
### RStudio Project vs. active usethis project {#sec-rstudio-project-vs-active-usethis-project}
```

Spidey sense and/or aesthetics tells me to avoid the `.` here.

Reply:
Definitely - that was just a typo propagated by cut and paste!